lib/bundle.js

```diff
@@ -103,31 +199,33 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     return systemBuilder.trace(firstModule, cfg);
```

Comment:
If we did this, the globbing doesn't need to alter the bundle function - it can actually be done as a `globExpression` function which converts a globbed module expression into a non-globbed module expression.

For example:
```
  module/a + lib/**/* - lib/*
```
We then glob each expression individually and expand it in place:
```
  module/a + lib/sub/path + lib/a + lib/b - lib/b - lib/a
```

We don't need to worry about arithmetic reduction though.

Not saying it has to be this way, just throwing out ideas for making it consistent. Let me know what you think.